SConstruct

```diff
@@ -95,7 +95,7 @@ env_base.SConsignFile(".sconsign{0}.dblite".format(pickle.HIGHEST_PROTOCOL))
 
 customs = ["custom.py"]
 
-profile = ARGUMENTS.get("profile", False)
+profile = methods.get_cmdline_bool("profile", False)
```

Comment:
Profile is a boolean? We use it in line 100 as a string.

Reply:
You're right, I just saw the `False` fallback value and assumed it's a bool. Feel free to send a PR to change that back, or I'll do it tomorrow.